src/bandersnatch_storage_plugins/s3.py

```diff
@@ -0,0 +1,413 @@
+#  type: ignore
```

Comment:
This feels a bit dangerous, is there a better way of suppressing the invalid / "fix-later" type errors than this?

Reply:
There're too many type error in this file, I have to suppress the whole file or I have to suppress every method in S3Storage, it will be automatically fixed if s3path has full type hint for every method.